dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/DotFolderTransformerBuilder.java

```diff
@@ -0,0 +1,103 @@
+package com.dotmarketing.portlets.contentlet.transform;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.folders.business.FolderAPI;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Builder to gain access to the Folder Transformer
+ */
+public class DotFolderTransformerBuilder {
+
+    private FolderAPI folderAPI = APILocator.getFolderAPI();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'folderAPI' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)